src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -87,6 +87,9 @@ function transform(node) {
 		extractedAttributes[attributeName] = attributeValue;
 	}
 	
+	if (node.checked) {
+		extractedAttributes["checked"] = node.checked;
+	}
 	// overwrite empty attributes (e.g. 'disabled')
 	extractedAttributes["disabled"] = isDisabled(node);
 	extractedAttributes["read-only"] = node.readOnly;
```

Comment:
I just noticed... We should take the names from their respective attributes: html - `readonly`

Reply:
Separate PR ;-)

Reply:
Sure, just to put it to your attention.